cli/pkg/artifact_builder/artifact_builder.go

```diff
@@ -53,7 +57,7 @@ func (ab ArtifactBuilder) WithBackend(backend *backend.Backend) ArtifactBuilderI
 
 func (ab ArtifactBuilder) WithTags(tags []string) ArtifactBuilderIface {
 	if len(tags) > 0 {
-		ab.tags = tags
+		ab.Tags = tags
```

Comment:
As you can see, `WithTags()` becomes redundant as `Tags` can be accessed directly. Or through `SetTags()`.